go.mod

```diff
@@ -3,6 +3,7 @@ module go.etcd.io/bbolt
 go 1.21
 
 require (
+	github.com/fuweid/go-dmflakey v0.1.0
```

Comment:
Many thanks to @fuweid for putting the package together.  I am happy with either bringing it under `/test` given it is specific to `bbolt` within etcd project currently.  If we proceed with that I think it would be a good test of nested `OWNERS` files to give @fuweid approver access for pr's to that code.

Reply:
Updated